scripts/edit-datapackage-json.py

```diff
@@ -1,9 +1,14 @@
 from pathlib import Path
-from pprint import pprint
 
 import seedcase_sprout.core as sp
 
-properties = sp.PackageProperties(
+package_path = Path(__file__).resolve().parent.parent
```

Comment:
This way of coding could be something we try to improve. But something to keep in mind as we develop it more.